evm_loader/program/src/account_storage.rs

```diff
@@ -243,8 +242,22 @@ impl<'a> ProgramAccountStorage<'a> {
                         debug_print!("Apply can't be done. Not found account for address = {:?}.", address);
                         return Err(ProgramError::NotEnoughAccountKeys);
                     }
+                },
+                Apply::Delete { address } => {
+                    debug_print!("Going to delete address = {:?}.", address);
+
+                    if let Some(pos) = self.find_account(&address) {
+                        let account_info = &self.account_metas[pos];
+                        debug_print!("Move funds from account");
+                        **account_info.lamports.borrow_mut() = 0;
+                        debug_print!("Mark account empty");
+                        let mut account_data = account_info.try_borrow_mut_data()?;
+                        AccountData::pack(&AccountData::Empty, &mut account_data)?;
```

Comment:
Can you remove Solana account instead of writing the `AccountData::Empty` value?

Reply:
No, account still alive until next block.